blueman/plugins/applet/PPPSupport.py

```diff
@@ -1,19 +1,28 @@
 from gettext import gettext as _
+from typing import TYPE_CHECKING, Callable, Union
 
+from _blueman import RFCOMMError
+
+from blueman.Service import Service
 from blueman.plugins.AppletPlugin import AppletPlugin
 from blueman.gui.Notification import Notification
 from blueman.main.DBusProxies import Mechanism
 from blueman.main.Config import Config
 
 from gi.repository import GLib
 
-from blueman.Sdp import DIALUP_NET_SVCLASS_ID
 import subprocess
 import logging
 
+from blueman.services import DialupNetwork
+
+if TYPE_CHECKING:
+    from blueman.main.Applet import BluemanApplet
```

Comment:
What is the problem with always importing it?

Reply:
Circular imports.